### ISSUE TITLE
Rate limit exceeded when building the documentation in CI

### ISSUE THREAD

**Issue by pfultz2** [ROCm/rocm-docs-core#919]
Our CI for migraphx fails when building the documentation, from a timeout error. It looks like its trying to fetch things from the internet and then we it a rate limit that then never finishes causing the job to timeout. This is the output from the logs:

```
Remote mappings will be fetched from ROCm/rocm-docs-core branch=develop
Request GET /repos/ROCm/rocm-docs-core failed with 403: rate limit exceeded
Setting next backoff to 2500.125815s
Request GET /repos/ROCm/rocm-docs-core failed with 403: rate limit exceeded
Setting next backoff to 938.828504s
Request GET /repos/ROCm/rocm-docs-core failed with 403: rate limit exceeded
Setting next backoff to 1446.826964s
Request GET /repos/ROCm/rocm-docs-core failed with 403: rate limit exceeded
Setting next backoff to 2272.82957s
```

Is there way to either disable it downloading from the internet? Or fallback on something when it hits a 403 or 404?

It seems like documentation should be able to be built completely offline without needing an internet connection.

**Comment by samjwu** [comment]
Yes, overwriting the external_projects_remote_repository in the conf.py will disable fetching the projects.yaml, which is what tells intersphinx where to download inventory files

`external_projects_remote_repository = ""`

**Comment by samjwu** [comment]
example https://github.com/ROCm/rocm-docs-core/blob/develop/docs/conf.py#L13
